python/nav/web/templates/foundation-5/field.html

```diff
@@ -1,46 +1,41 @@
-{% load crispy_forms_field %}
-
 {% if field.is_hidden %}
   {{ field }}
 {% else %}
   <div id="div_{{ field.auto_id }}"
-       class="ctrlHolder
-       {% if field.errors and form_show_errors %} error{% endif %}
-       {% if field|is_checkbox %} checkbox{% endif %}
-       {% if field.css_classes %} {{ field.css_classes }}{% endif %}">
+       class="ctrlHolder{% if field.errors %} error{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}"
+       style="background: #BADA55 !important; padding: 4px !important;"
+  >
     {% spaceless %}
 
       {% if field.label %}
-        {% if field|is_checkbox %}
-          {% crispy_field field %}
+        {% if field.field.widget.input_type == 'checkbox' %}
+          {{ field }}
         {% endif %}
 
         <label for="{{ field.id_for_label }}"
                {% if field.field.required %}class="requiredField"{% endif %}>
-          {{ field.label|safe }}{% if field.field.required %}
-            <span class="asteriskField">*</span>{% endif %}
+          {{ field.label|safe }}
+          {% if field.field.required %}
+            <span class="asteriskField">*</span>
+          {% endif %}
           {% if field.help_text %}
-            &nbsp;<i id="hint_{{ field.auto_id }}" class="fa fa-info-circle right"
-                     data-tooltip title="
-                  {{ field.help_text }}
-                  "></i>
+            <i id="hint_{{ field.auto_id }}" class="fa fa-info-circle right"
+               data-tooltip title="{{ field.help_text }}"></i>
```

Comment:
AFAICT, this is just a re-indentation of the same HTML, right? (except the non-breaking space is gone...)